### PR TITLE
Query parameter override

### PR DESCRIPTION
**Pull request by Nagriar**
We should not send the query parameter with the default value because they can be set in the index settings but It'll change the behavior with the previous version, @redox What do you think about it?

### CODE REVIEW

File: src/main/java/com/algolia/search/saas/Query.java [hunk 1]
```diff
@@ -31,1002 +31,1086 @@
...
+	 * @return the advancedSyntax
+	 */
+	public Boolean isAdvancedSyntax() {
+		return advancedSyntax;
+	}
+
+	/**
+	 * @return the page
+	 */
+	public
```
Comment (bobylito) [inline src/main/java/com/algolia/search/saas/Query.java:979]: OUPS

**Comment by Nagriar** [comment]
Thanks @bobylito, It should be easier to read now :)

**Comment by bobylito** [comment]
It looks good to me :)

**Comment by redox** [comment]
:+1: LGTM @Nagriar

**Pull request merged**